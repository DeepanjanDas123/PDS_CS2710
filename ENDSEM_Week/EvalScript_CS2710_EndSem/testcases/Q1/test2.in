BuildFrequencyTable
VLTg9LTVR
Quit
