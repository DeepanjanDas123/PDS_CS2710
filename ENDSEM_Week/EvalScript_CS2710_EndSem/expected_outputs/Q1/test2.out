9 1
L 2
R 1
T 2
V 2
g 1
